library/src/main/java/com/schibsted/spain/barista/BaristaMenuClickActions.java

```diff
@@ -0,0 +1,54 @@
+package com.schibsted.spain.barista;
+
+import android.support.annotation.IdRes;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.espresso.NoMatchingViewException;
+
+import static android.support.test.espresso.Espresso.onData;
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.Espresso.openActionBarOverflowOrOptionsMenu;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.matcher.RootMatchers.isPlatformPopup;
+import static com.schibsted.spain.barista.custom.DisplayedMatchers.displayedWithDescription;
+import static com.schibsted.spain.barista.custom.DisplayedMatchers.displayedWithId;
+import static com.schibsted.spain.barista.custom.DisplayedMatchers.displayedWithText;
+import static com.schibsted.spain.barista.custom.HelperMatchers.menuMatcher;
+import static org.hamcrest.Matchers.hasToString;
+import static org.hamcrest.core.Is.is;
+
+public class BaristaMenuClickActions {
+
+  public static void menuClick(@IdRes int id) {
```

Comment:
Why not starting the method name with a verb? It will meet the same style as the rest of the project.